src/ai/backend/common/clients/valkey_client/valkey_container_log/client.py

```diff
@@ -0,0 +1,158 @@
+import logging
+from typing import (
+    Optional,
+    Self,
+)
+
+from glide import (
+    Batch,
+)
+
+from ai.backend.common.clients.valkey_client.client import (
+    AbstractValkeyClient,
+    create_layer_aware_valkey_decorator,
+    create_valkey_client,
+)
+from ai.backend.common.log.types import ContainerLogData
+from ai.backend.common.metrics.metric import LayerType
+from ai.backend.common.types import ValkeyTarget
+from ai.backend.logging.utils import BraceStyleAdapter
+
+log = BraceStyleAdapter(logging.getLogger(__spec__.name))
+
+# Layer-specific decorator for valkey_container_log client
+valkey_decorator = create_layer_aware_valkey_decorator(LayerType.VALKEY_CONTAINER_LOG)
+
+
+class ValkeyContainerLogClient:
+    """
+    Client for interacting with Valkey for container log operations using GlideClient.
```

Comment:
Please add an explicit description about how this class differs from the vanilla `ValkeyClient`.

```suggestion
    Client for interacting with Valkey for container log operations using GlideClient.

    This client intentionally ignores server-side failures or connection failures in
    its log-related action methods so that crashes or failures of the container log
    subsystem would not impact the other parts of the system.
```

Reply:
Actually, now the logic is in caller side. to skip to call brocast logic, when the operation is failed.

Reply:
@achimnol applied you review.